verifier/src/lib.rs

```diff
@@ -10,67 +10,72 @@ pub use report_body::{
     MiscellaneousSelectVerifier, MrEnclaveVerifier, MrSignerVerifier, ReportDataVerifier,
 };
 
-use core::fmt::Debug;
+use core::fmt::{Debug, Display, Formatter};
 use mc_sgx_core_types::{
     Attributes, ConfigId, ConfigSvn, IsvSvn, MiscellaneousSelect, MrEnclave, MrSigner, ReportData,
 };
 use subtle::CtOption;
 
+/// An error that implements the [`Display`] trait.
+pub trait DisplayableError: Display + Clone {}
```

Comment:
Needed to have `Clone` here as `CtOption` consumes itself anytime one wants to get the underlying value.